### DISCUSSION TITLE
Some Feature Request

### DISCUSSION THREAD

**Discussion by ty114514** [lollipopkit/flutter_server_box#71] [Ideas]
+ 类似juicessh，音量键控制终端分辨率（有时候界面太小显示不全）
+ 长按方向键快速移动光标，字多一个个点手麻
+ 类似termius可以自定义终端字体颜色
+ 键盘上方按钮添加pageup pagedown，支持点击时震动，点击的时候无论滚动条在哪立刻回到最下方

**Comment by lollipopkit** [comment]
👌

第一个可能不行，在询问第三方插件是否可行
第二个已完成
第三个可能会导致显示很杂，暂不考虑
第四个会做，在考虑自定义按键

但我还是想说 `Alt + 方向键` 可以以单词为单位移动光标（.....无恶意，只是我不想动手加这个功能）
